server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -201,10 +201,25 @@ Feature: GetMessages method
     And the list should contain 1 message
     And the textBody of the message is "Hello text body\n"
     And the htmlBody of the message is "<html>Hello html body</html>\n"
-    
+
 Scenario: Retrieving message with more than 1000 char by line should return message when exists
     Given the user has a message "m1" in "inbox" mailbox beginning by a long line
     When the user ask for messages "m1"
     Then no error is returned
     And the list should contain 1 message
     And the id of the message is "m1"
+
+Scenario: Retrieving message which contains multiple same inlined attachments
+    Given the user has a message "m1" in the "inbox" with multiple same inlined attachments "ia1"
+    When the user ask for messages "m1"
+    Then no error is returned
+    And the list should contain 1 message
+    And the hasAttachment of the message is "true"
+    And the list of attachments of the message contains 1 attachments
+    And the first attachment is:
```

Comment:
the eml is an implementation detail : you should be able to understand the whole test by reading the gherkin file. Here, the values you require are not described earlier in the gherkin and I think that's a problem.